### PR TITLE
gkmSVM initial commit

### PR DESCRIPTION
**Pull request by visze**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

gkmSVM: Gapped-Kmer Support Vector Machine from CRAN for modeling the properties and functions of DNA and protein sequence.

citations for the software:
- Ghandi M, Lee D, Mohammad-Noori M, Beer MA. 2014. Enhanced Regulatory Sequence Prediction Using Gapped k-mer Features. PLoS Comput Biol 10: e1003711.
- Ghandi M, Mohammad-Noori M, Gharegani N, Lee D, Garraway L, Beer MA. 2016. gkmSVM: an R package for gapped-kmer SVM. Bioinformatics 32: 2205-2207.

### CODE REVIEW

**Comment by visze** [comment]
I am only aware of bio* use cases (like deltaSVM) Not of any other. Gapped
kmer svm is also know in the bio* field and not in others. So for me it
makes no sense to release it in conda-forge

Am 22.11.2017 10:18 schrieb "Björn Grüning" <notifications@github.com>:

**Comment by daler** [comment]
@visze @bgruening I do think it belongs in bioconda. It has bioconductor dependencies and the original tool is published in Bioinformatics.

@visze would you please remove all the comments from the build and meta.yaml?

**Comment by visze** [comment]
@daler thx for you comments. I removed the unnecessary comments from the two files

File: recipes/r-gkmsvm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  license: GPL (>= 2)
+  summary: 'Imports the ''gkmSVM'' v2.0 functionalities into R (www.beerlab.org/gkmsvm). It also
+    uses the ''kernlab'' library (separate R package by different authors) for various
+    SVM algorithms.   '
+
+  license_family: GPL3
+
+# The original CRAN metadata for this package was:
+
+# Package: gkmSVM
```
Comment (bgruening) [inline recipes/r-gkmsvm/meta.yaml]: @visze here are some more comments that could be removed.

Comment (visze) [reply recipes/r-gkmsvm/meta.yaml]: yep I thought this might be helpful... but of course I can remove them

**Comment by bgruening** [comment]
Ok, thanks for the explanation. Looks good to me.

**Pull request merged**